packages/opentelemetry-plugin-document-load/src/documentLoad.ts

```diff
@@ -51,6 +51,22 @@ export class DocumentLoad extends BasePlugin<unknown> {
     });
   }
 
+  /**
+   * Adds spans for all resources
+   * @param rootSpan
+   * @private
```

Comment:
Would it make sense to remove the `@private` since it's redundant with the TS `private`? Or is there some value here in the JSDoc I'm missing?

Reply:
yes definitely, I just forgot to remove it after auto generated from my IDE